frontend/src/Editor/QueryManager/Components/QueryManagerBody.jsx

```diff
@@ -64,8 +65,11 @@ export const QueryManagerBody = forwardRef(
     */
     const [selectedQueryId, setSelectedQueryId] = useState(selectedQuery?.id);
 
+    const autoUpdateDataQuery = debounce(updateDataQuery, 500);
```

Comment:
It can be removed if we're not using it. Same with the unused props, state and variables in this component